### PR TITLE
D3ASIM-230 new select component

### PR DESCRIPTION
**Pull request by Lelith**
**Who**: @lelith

**What / Why**:  Component for native selects. The native select has many advantages instead of a handknitted list that simulated dropdown behaviour. Its more stable and has many advantages on mobile / touch devices. The contra is that we can not follow exactly the design with a styled open list. But i would put more weight in usability advantages, security and stability than in applying the design.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,13 +1,14 @@
 {
   "name": "gsy-component-library",
-  "version": "0.2.7",
+  "version": "0.2.8",
   "description": "reusable components from grid singularity",
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
+    "imagemin": "imagemin ./lib/images/* --out-dir=build/images",
```
Comment (cfleschhut) [inline package.json]: @Lelith Minor: Looks like `imagemin` needs to be added to our `devDependencies`?

**Comment by cfleschhut** [comment]
@Lelith Code LGTM, only the dropdown styling looks a bit off to me (mainly the icon vertical alignment)

localhost_9001_iframe html_selectedkind d3a 2fatoms selectedstory native 20select

File: package.json (hunk 1 above)
Comment (Lelith) [reply package.json]: will add

**Comment by cfleschhut** [comment]
@Lelith @eleventhirtyfour Btw, to create the screenshot above I used "Capture node screenshot" in Chrome DevTools:
https://developers.google.com/web/updates/2017/08/devtools-release-notes#node-screenshots
👍

**Pull request merged**